oms-monitor/docker/logstash/pipeline/logstash.conf

```diff
@@ -79,28 +79,26 @@ filter {
 
 
 ## From logspout: our containers' output
```

Comment:
this block as a whole would be removed later in a future, once I'll switch all the services to log as json